charts/apisix/values.yaml

```diff
@@ -96,6 +96,16 @@ apisix:
   podAntiAffinity:
     enabled: false
 
+  # timezone is the timezone where apisix uses.
+  # For example: "UTC" or "Asia/Shanghai"
+  # This value will be set on apisix container's environment variable TZ.
+  # You may need to set the timezone to be consistent with your local time zone,
+  # otherwise the apisix's logs may used to retrieve event maybe in wrong timezone.
+  timezone: UTC
+
+  # extraEnvVars An array to add extra env vars
+  extraEnvVars: []
```

Comment:
The format for the key and value should be explained.

Reply:
Done! PTAL.

Reply:
Shall we also add it in the README?